src/src/PluginStructs/P027_data_struct.h

```diff
@@ -11,6 +11,18 @@
 # define INA219_ADDRESS2                        (0x41) // 1000001 (A0=VS+, A1=GND)
 # define INA219_ADDRESS3                        (0x44) // 1000100 (A0=GND, A1=VS+)
 # define INA219_ADDRESS4                        (0x45) // 1000101 (A0=VS+, A1=VS+)
+# define INA219_ADDRESS5                        (0x42) // 1000010 (A0=SDA, A1=GND)
+# define INA219_ADDRESS6                        (0x43) // 1000011 (A0=SCL, A1=GND)
+# define INA219_ADDRESS7                        (0x46) // 1000110 (A0=SDA, A1=VS+)
+# define INA219_ADDRESS8                        (0x47) // 1000111 (A0=SCL, A1=VS+)
+# define INA219_ADDRESS9                        (0x48) // 1001000 (A0=GND, A1=SDA)
+# define INA219_ADDRESS10                       (0x49) // 1001001 (A0=VS+, A1=SDA)
+# define INA219_ADDRESS11                       (0x4A) // 1001010 (A0=SDA, A1=SDA)
+# define INA219_ADDRESS12                       (0x4B) // 1001011 (A0=SCL, A1=SDA)
+# define INA219_ADDRESS13                       (0x4C) // 1001100 (A0=GND, A1=SCL)
+# define INA219_ADDRESS14                       (0x4D) // 1001101 (A0=VS+, A1=SCL)
+# define INA219_ADDRESS15                       (0x4E) // 1001110 (A0=SDA, A1=SCL)
+# define INA219_ADDRESS16                       (0x4F) // 1001111 (A0=SCL, A1=SCL)
```

Comment:
Any specific reason why the addresses are out of (numerical) order?
They will also be out of order in the combobox of the config page.

Reply:
I kept the original 4 selections in their original order, and added the 'unusual' addresses after that set.

If you think it's better to use the numerical order, that's no issue, I can change that

Reply:
Well you store the value, so even if later someone asks for a more logical order, it doesn't need any transition in the settings file.

Reply:
I'll re-order it now, only to avoid another PR 😄